data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,272 @@
+			`	They take you into the cave, finally showing the ship they were refering to. "This is a mighty relic of the past, from the times our Empire used to control the stars; a Ladybug!`
+			scene "thumbnail/hai ladybug"
+			`	"Of course, it has some... drawbacks, compared to its golden time, and don't expect the best price in a shipyard either. Nevertheless it is a fine vessel, capable of safely taking a respectable amount of cargo across large distances! So, what do you say, <last>? This is the opportunity of a lifetime, as they simply don't make ships like these anymore!"`
+			choice
+				`	"I will take this deal."`
+					goto take
+				`	"I am not interest in this deal."`
+			`	"A shame, this probably means we will not meet again," they say, before leaving, as discreetly as they came.`
+				decline
+			label take
+			action
+				give ship "Ancient Ladybug" "Good Omen"
+				fine 6000000
+			`	They give you access to the ship and proceed to explain the details. "It is of no real significance, but surely you understand we cannot go trough official means for this, we will have to present it as a fine. Worry not, we will make it something of no consequence, like an old Korath forbidden tech you would not have known about, that we had to confiscate."`
+			`	You quickly realize they are not asking you for your opinion at all. In fact, you notice the fine has already been issued before you could say anything. "Oh, and you should not change its colors, you'll understand why later."`
+
+mission "Young but not that young"
+	landing
+	description "Use the newly acquired old Ladybug to bring aid to the Unfettered Hai."
+	deadline 20
+	cargo "food (aid)" 121
+	to offer
+		has "ship model: Ancient Ladybug"
+	on enter
+		system
+			attributes "unfettered"
+		"reputation: Hai (Unfettered)" >?= 0
+		dialog `As you enter the system you notice something... Or, more precisely, the lack of something. It is too quiet, probably due to the fact that, for the first time, the Unfettered Hai are not shooting at you. They are not hailing you either, however. Perhaps they think you are one of the Hai aiding them, especially given your cargo.`
+	on fail
+		"reputation: Hai (Unfettered)" <?= -1000
+	on complete
+		dialog `You receive a message from the 'associates of Hai-home', "thank you for helping our brethren, <last>. We will not forget this. Please accept these <payment> to get you started on paying back."`
+		payment 1250000
+
+
+
+outfit "Unfettered Landing License"
+	category "Licenses"
+	thumbnail "outfit/unknown"
+	"maintenance costs" 600
+	description "This is needed in order to land on Unfettered planet, but can be sold at any time in order to cancel the arrangement."
```

Comment:
You can't sell licenses.

Reply:
Pretty sure you can if its been given like this

Reply:
It will have to be tested, at least.

Reply:
Yeah everything will be
It could use an image as well, or maybe a nice angle on a ladybug could work

Reply:
I can confirm you can sell it indeed.